src/sentry/api/endpoints/organization_auth_providers.py

```diff
@@ -0,0 +1,32 @@
+from __future__ import absolute_import
+
+from rest_framework.response import Response
+
+from sentry import features
+from sentry.auth import manager
+from sentry.auth.providers.saml2 import SAML2Provider, HAS_SAML2
+from sentry.api.bases.organization import OrganizationEndpoint, OrganizationAdminPermission
+from sentry.api.serializers import serialize
+
+
+class OrganizationAuthProvidersEndpoint(OrganizationEndpoint):
+    permission_classes = (OrganizationAdminPermission, )
+
+    def get(self, request, organization):
+        """
+        List available auth providers that are available to use for an Organization
+        ```````````````````````````````````````````````````````````````````````````
+
+        :pparam string organization_slug: the organization short name
```

Comment:
Is `pparam` a special parameter marking? I've usually seen `param`.

Reply:
hmm it's probably a typo

Reply:
Nope, it seems to be used everywhere

Reply:
Yeah, this is a typo.